plugin/pkg/scheduler/factory/factory.go

```diff
@@ -147,6 +157,84 @@ func NewConfigFactory(client *client.Client, schedulerName string, hardPodAffini
 		},
 	)
 
+	c.PVLister.Store, c.pvPopulator = cache.NewInformer(
+		c.createPersistentVolumeLW(),
+		&api.PersistentVolume{},
+		0,
+		cache.ResourceEventHandlerFuncs{
```

Comment:
For each of those, please add a comment why we are clearning the cache (i.e. what predicate is invalidated).